docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -75,11 +75,10 @@ services:
     networks:
       - stargate
     ports:
-      - 9080:9080
+      - 8085:8080
     mem_limit: 2G
     environment:
       - STARGATE_BRIDGE_HOST=coordinator
       - STARGATE_BRIDGE_PORT=8091
-      - STARGATE_GRAPHQL_PORT=9080
```

Comment:
We actually don't need to customize the port since the API is in its own container.

Reply:
you can still set the exposed port with `-Dquarkus.http.port` if needed..